README.md

```diff
@@ -59,6 +59,18 @@ modifiers:
   #  custom_detok_trg: zh
   #  # template: " <tag{n}> {token} </tag{n}>" # This is the default way of inserting tags. Beware of changing it.
                                                # DO NOT include it in the config as it's a default parameter.
+# - Typos: 0.05 # Modify 5% of the input sentences to contain plausible typos.
```

Comment:
Unclear to me from this description. If 0.05% of the sentences are affected, than how many typos would be applied to the words in the "chosen" sentence.